core/domain/email_manager.py

```diff
@@ -248,7 +248,7 @@ def log_new_error(*args, **kwargs):
 }
```

Comment:
Why did the file permissions change? Please revert.

Reply:
Also same for the other file.

Reply:
We found that we accidentally added an executable permission to the files. Not sure how that happened, but we went ahead and removed it for both files. The file permissions should be reverted now.